app/router.js

```diff
@@ -8,6 +8,9 @@ const Router = Ember.Router.extend({
 Router.map(function() {
   this.route('about');
   this.route('contact');
+  this.route('rentals', function() {
+    this.route('show', { path: '/:title' });
```

Comment:
I like the slug approach as most API's would have one available. Should that be addressed in a separate PR?